doc/api/assert.md

```diff
@@ -18,6 +18,9 @@ An alias of [`assert.ok()`][].
 <!-- YAML
 added: v0.1.21
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/12142
+    description: Sets and Maps are handled correctly.
```

Comment:
"handled correctly" is a bit vague..maybe "elements in Sets and Maps are compared during deep comparison" or something like that?